src/core/currency/wallet/currency-wallet-export.js

```diff
@@ -24,6 +24,48 @@ export function dateFilter(
   return out
 }
 
+export function searchStringFilter(
+  txs: EdgeTransaction[],
+  opts: EdgeGetTransactionsOptions
+): EdgeTransaction[] {
+  const { searchString } = opts
+
+  let out = txs
```

Comment:
normalize all search strings and transaction information to lowercase Before search to make it case insensitive